backend/src/schema/types/type/User.gql

```diff
@@ -56,14 +56,14 @@ type User {
   contributionsCount: Int! @cypher(
     statement: """
       MATCH (this)-[:WROTE]->(r:Post)
-      WHERE (NOT exists(r.deleted) OR r.deleted = false)
-      AND (NOT exists(r.disabled) OR r.disabled = false)
+      WHERE NOT r.deleted = true AND NOT r.disabled = true
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-06T23:47:54Z" title="Friday, June 7th 2019, 1:47:54 am +02:00">Jun 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -56,14 +56,14 @@ type User {
   contributionsCount: Int! @cypher(
     statement: """
       MATCH (this)-[:WROTE]->(r:Post)
-      WHERE (NOT exists(r.deleted) OR r.deleted = false)
-      AND (NOT exists(r.disabled) OR r.disabled = false)
+      WHERE NOT r.deleted = true AND NOT r.disabled = true
```


@ulfgebhardt there was a strange bug that the pagination button ("Fetch more") would not go away even though there are no records left. This is the reason why:

![Screenshot - 2019-06-07T014505 476](https://user-images.githubusercontent.com/2110676/59073083-ed23ec00-88c5-11e9-8e90-b2155e1df4d0.png)

Pagination on this page takes into account the total number of `contributionsCount` which you've overwritten in 6f0447515. It was incorrectly more than expected, the condition `records.length == total` never met and pagination wouldn't disappear.